src/main/java/com/godev/linkhubservice/rest/controllers/AccountController.java

```diff
@@ -35,4 +33,7 @@ public interface AccountController {
     })
     @PostMapping(value = "/auth")
     ResponseEntity<AuthResponse> auth(@RequestBody @Valid AuthRequest authRequest);
+
+    @PutMapping
+    ResponseEntity<UpdateAccountResponse> update(@Valid @RequestBody UpdateAccountRequest updateAccountRequest);
```

Comment:
Arrange annotation order @Valid @RequestBody

Reply:
Check! Did you see that in the register method the order of these annotations is also changed? Should I exchange them too?